overlord/servicestate/quota_handlers.go

```diff
@@ -221,6 +223,94 @@ func addRestartServicesTasks(st *state.State, queueTask func(task *state.Task),
 	}
 }
 
+func AddSnapToQuotaGroup(st *state.State, snapName string, quotaGroup string) (*state.Task, error) {
```

Comment:
this also belongs to quota_control.go, it's not a handler

Reply:
this is not unit tested here

Reply:
Moved this into quota_control and added unit tests